lib/wpMandrill.class.php

```diff
@@ -38,24 +38,40 @@ static function on_load() {
 
             function wp_mail( $to, $subject, $message, $headers = '', $attachments = array() ) {
                 try {
-                    $sent = wpMandrill::mail( $to, $subject, $message, $headers, $attachments );
-
-                    if (    is_wp_error($sent)
-                        || !isset($sent[0]['status'])
-                        || ($sent[0]['status'] != 'sent' && $sent[0]['status'] != 'queued') ) {
-
-                        do_action( 'wp_mail_native', $to, $subject, $message, $headers, $attachments );
-                    }
-
-                    return true;
+                    $response = wpMandrill::mail( $to, $subject, $message, $headers, $attachments );
+                    self::evaluate_response( $response );
```

Comment:
This upgrade broke the checkout process in our WooCommerce store (sending email on new order). Here is an isolated version of this code that hopefully highlights the issue https://3v4l.org/4ZZn9

So calling `on_load` exposes the `wp_mail` function but not in the class scope, so when you subsequently call it, it cannot access `self::evaluate_response()`.

Reply:
We're looking into it.